docs/config-file/v2.rst

```diff
@@ -175,6 +175,29 @@ Example:
   manage the build, this setting will not have any effect. Instead
   add the extra requirements to the ``environment`` file of Conda.
 
+Packages list
+'''''''''''''
+
+Install packages from a list of requirements. This allows you to embed requirements in the
+``.readthedocs.yml`` configuration directly.
```

Comment:
```suggestion
Install packages from a list of requirements.
This allows you to declare dependencies in the ``.readthedocs.yml`` file instead of creating a separate file.
```